format/FlightSql.proto

```diff
@@ -1064,8 +1064,8 @@ enum Searchable {
  * The returned schema will be:
  * <
  *   type_name: utf8 not null (The name of the data type, for example: VARCHAR, INTEGER, etc),
- *   data_type: int not null (The SQL data type),
- *   column_size: int (The maximum size supported by that column.
+ *   data_type: int32 not null (The SQL data type),
+ *   column_size: int32 (The maximum size supported by that column.
  *                     In case of exact numeric types, this represents the maximum precision.
  *                     In case of string types, this represents the character length.
  *                     In case of datetime data types, this represents the length in characters of the string representation.
```

Comment:
Could you add 2 spaces indent to corresponding documents too?

Reply:
Sure! Updated